### DISCUSSION TITLE
Are there any options in joint js to lock the diagram

### DISCUSSION THREAD

**Discussion by reshin-pulapaddy** [clientIO/joint#1936] [Q&A]
Are there any options in joint js to lock the diagram

**Comment by jamesgeorgewilliams** [comment]
If by "lock", you mean that interactions with links and elements are disabled, the paper has an `interactive` option.
https://resources.jointjs.com/docs/jointjs/v3.6/joint.html#dia.Paper.prototype.options.interactive

You could also create a "View" and "Edit" mode for diagrams as seen in the following demo:
https://codepen.io/jointjs/pen/ExRvvGb
https://github.com/clientIO/joint/discussions/1895

**Reply by reshin-pulapaddy** [reply]
Thank @jamesgeorgewilliams
